### PR TITLE
doc: SRT administrative processes

### PR DESCRIPTION
**Pull request by frasertweedale**
Document various administrative processes within the SRT.

### CODE REVIEW

**Review by mihaimaruseac — APPROVED** [review]
Thank you

File: docs/membership.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  needed.
+
+- Publish the call on [Discourse] and promote it in the logical ways
+  (e.g. Haskell Foundation social media, r/haskell, etc).
+
+- The application period should be about 4 weeks.  Bump and do
+  another burst of promotion at the halfway point.
+
+- After the application deadline, the full SRT membership reviews
+  the proposals and selects the new member(s).
```
Comment (TristanCacqueray) [inline docs/membership.md:68]: Would be nice to have a formal process here. Last time we used a mail thread, and I'd like us to consider a proper voting system.

**Review by TristanCacqueray — APPROVED** [review]
Looks great, thanks @frasertweedale !

**Review by blackheaven — APPROVED** [review]

File: docs/membership.md (hunk 1 above)
Comment (frasertweedale) [reply docs/membership.md:68]: Let us plan more thoroughly ahead of the next call for volunteers (whenever that may be).

**Comment by frasertweedale** [comment]
Thanks for the reviews.  Merging.

**Pull request merged**